test/parallel/test-http-server-unconsume.js

```diff
@@ -4,14 +4,18 @@ const assert = require('assert');
 const http = require('http');
 const net = require('net');
 
-let received = '';
+let received1 = '';
+let received2 = '';
 
 const server = http.createServer(function(req, res) {
   res.writeHead(200);
   res.end();
 
   req.socket.on('data', function(data) {
-    received += data;
+    received1 += data;
+  });
+  req.socket.addListener('data', function(data) {
```

Comment:
This doesn't quite test the correct thing because the first listener already triggers the change we're looking for. Luckily, the fix is simple — you can just do `assert.strictEqual(req.socket.on, req.socket.addListener);` on line 13 and that'll be enough.

Reply:
done